src/services/ApiService.ts

```diff
@@ -505,4 +505,13 @@ export class ApiService {
             }
         }
     };
+
+    public getSimbad = (apiAddress: string, query: string, cancelTokenSource?: CancelTokenSource): Promise<AxiosResponse<any>> => {
```

Comment:
move into separate class. `ApiService` is forinteracting with the CARTA backend/server API. We can rename this class if you want